CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+Want to contribute? Great! First, read this page.
+
+# If there is no issue, open one
+Check if what you want to add/fix is documented in an issue. If not open one. Then, you can clone the repository and start hacking!
```

Comment:
Should be an option to skip an issue of the change is trivial.

Reply:
absolutely not, only admins should push to master